master/static/migrations/20211028145058_jobs.up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TYPE public.job_type AS ENUM (
+    'EXPERIMENT',
+    'NOTEBOOK',
+    'SHELL',
+    'COMMAND',
+    'TENSORBOARD'
+);
+
+CREATE TABLE public.jobs (
+    job_id text NOT NULL UNIQUE,
+    job_type job_type NOT NULL
+);
+
+
+BEGIN;
+
+INSERT INTO jobs (
+    SELECT 'backfilled-' || id as job_id, 'EXPERIMENT' as job_type FROM experiments
+);
+
+ALTER TABLE public.experiments
```

Comment:
i think this is fine, since the experiments table is probably small enough (<100k rows for sure)